<launch>
  <arg name="data_directory" default="/home/bianca.bendris/a4r_emb/data/" />
  <arg name="dir" default="20200618_163002_prob_from_wall" />
  <arg name="target_directory" default="$(arg data_directory)$(arg dir)" />
  <arg name="rosbag_file" default="$(arg data_directory)$(arg dir)/tmp_bag_2020-06-18-16-30-04.bag" />
  <arg name="rate" default="5"/>
  <arg name="mav_name" default="firefly"/>
  <arg name="visualize" default="true"/>

	<!-- Play Rosbag and RVIZ (visualization) -->
    <!-- <node pkg="rosbag" type="play" name="player" output="screen" args="-r $(arg rate) $(arg data_directory)/$(arg dir)/tmp_bag_2020-06-03-10-55-42.bag"/> -->
<!-- 
    <node name="eval_bag" pkg="active_3d_planning_app_reconstruction" type="eval_bag" output="screen" required="true">
        <remap from="odometry" to="/$(arg mav_name)/ground_truth/odometry" />
        <param name="target_directory" value="$(arg data_directory)$(arg dir)" />
        <param name="rate" value="1.0" />
    </node> -->

    <node name="eval_rosbag" pkg="active_3d_planning_app_reconstruction" type="eval_rosbag" output="screen" required="true">
        <param name="odometry_topic"  value="/$(arg mav_name)/ground_truth/odometry" />
        <param name="target_directory" value="$(arg target_directory)" />
        <param name="rosbag" value="$(arg rosbag_file)" />
    </node>


  <!-- RVIZ Visualization -->
  <!-- <node type="rviz" name="rviz_experiment" pkg="rviz" args="-d $(find active_3d_planning_app_reconstruction)/cfg/experiments/experiment_visualization.rviz" if="$(arg visualize)"/> -->
</launch>
